site/app/templates/admin/admin_gradeable/AdminGradeableCreate.twig

```diff
@@ -303,7 +303,7 @@
 </div>
 <div class="row">
     <input type="hidden" name="csrf_token" value="{{ csrf_token }}" />
-    <button class="btn btn-primary" type="submit" style="margin-left:10px;">Create New Gradeable</button>
+    <button class="btn btn-primary" type="submit" id="button" disabled="disabled" style="margin-left:10px;">Create New Gradeable</button>
```

Comment:
You should use a more descriptive id than `button`, something like `create-gradeable-btn` would work better so its easier to tell from the JS code what is being modified 

You do not need to set the disabled attribute to any value, just specifying `disabled` alone is enough